examples/permissioned-token/README.md

```diff
@@ -96,3 +96,50 @@ Developer hour links:
 - *implementation of the solution above using algob*: [examples/restricted-assets](/examples/restricted-assets)
 
 Link to spec of current template: [https://paper.dropbox.com/doc/Algob-Security-Token-Template-FR2LXhVg3edevYPBQZw6F](https://paper.dropbox.com/doc/Algob-Security-Token-Template-FR2LXhVg3edevYPBQZw6F)
+
+### API Doc
+Below we describe different use cases. We describe a use case using a function notation (along with transaction group structure). We implement that function in scripts, but for direct integration in smart contracts, you need to construct transactions as described below.
+
+#### Common
+
+1. [*totalSupply(assetIndex)*](examples/permissioned-token/scripts/common/common.js): returns total supply of the asset (*asset.total* - *asaReserveHolding.amount*).
```

Comment:
Please add a line how to get user token holding (this is a standard ASA query).

Reply:
ok